modules/exploits/linux/misc/quest_pmmasterd_bof.rb

```diff
@@ -50,41 +49,49 @@ def initialize(info = {})
         [
           ['Quest Privilege Manager pmmasterd 6.0.0-27 x64',
             {
-              :exploit => :exploit_x64,
-              :check => :check_x64
+              exploit: :exploit_x64,
+              check: :check_x64
             }
           ],
           ['Quest Privilege Manager pmmasterd 6.0.0-27 x86',
             {
-              :exploit => :exploit_x86,
-              :check => :check_x86
+              exploit: :exploit_x86,
+              check: :check_x86
             }
           ]
         ],
       'Privileged'     => true,
       'DisclosureDate' => 'Apr 09 2017',
       'DefaultTarget'  => 1
-      ))
-
-    register_options([ Opt::RPORT(12345) ], self.class)
-    register_options( [ Opt::CPORT(rand(1024))], self.class )
+      )
+    )
+
+    register_options(
+      [
+        Opt::RPORT(12345),
+        Opt::CPORT(rand(1024))
```

Comment:
not necessarily (see how docker images do it) - but I also don't know if it is necessary for the bug either